packages/dx-react-grid-bootstrap3/src/templates/table-header-cell.jsx

```diff
@@ -42,6 +42,10 @@ export class TableHeaderCell extends React.PureComponent {
       allowGroupingByClick, groupByColumn,
       allowDragging, dragPayload,
       allowResizing, changeColumnWidth, changeDraftColumnWidth,
+      changeSortingDirection,
+      // eslint-disable-next-line react/prop-types
```

Comment:
Is this disabling really needed?

Reply:
It prevents 'no-unused-vars' & 'react/prop-types' warnings.

Reply:
Ok. I understand. But seems that it may be a "good" solution to describe them in property types?

Reply:
It will be unused property. How users can use it?

Reply:
They shouldn't use it.

Reply:
Do you mean these props won't be documented?

Reply:
They are already documented. Isn't it?

Reply:
`tableRow` is in TableCellArgs, not in HeaderCellArgs. Agree in this case))